buildpipeline/pipeline.json

```diff
@@ -155,6 +155,24 @@
             "SubType": "netcoreapp"
           }
         },
+		{
+          "Name": "DotNet-CoreFx-Trusted-Windows",
```

Comment:
Why not instead of adding a new build definition for x86 use the one right before, delete the `PB_TargetQueue` and add the new `TargetQueue` to the `TargetQueues` list under `PB_CreateHelixArguments`, just as you did in the other build definitions?